tests/pages/firefox/base.py

```diff
@@ -2,17 +2,25 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+from pypom import Region
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support import expected_conditions as expected
 
 from pages.base import BasePage
-from pages.page import PageRegion
 
-# this should be enough to scroll elements so they are not beneath the header
-HEADER_OFFSET = -100
```

Comment:
We can actually be smarter about this now and offset by the height of the family navigation element.. I'll update with this change.